scripts/end-to-end.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
Is this a test helper? Then I think it should have "test" somewhere in its name.